chipy_org/settings.py

```diff
@@ -9,10 +9,11 @@
 def env_var(key, default=None):
     """Retrieves env vars and makes Python boolean replacements"""
     val = os.environ.get(key, default)
-    if val in ('True', 'true'):
-        val = True
-    elif val in ('False', 'false'):
-        val = False
+    if isinstance(val, str):
+        if val.lower() in ('t', 'true'):
+            val = True
+        elif val.lower() in ['f', 'false']:
```

Comment:
nit: ['f', 'false'] -> ('f', 'false') to align with line 13. 
super-nit: may be the lower() call can be a separate variable